doc/devel/documenting_mpl.rst

```diff
@@ -129,10 +128,10 @@ are:
    require multiple docstring edits.
 
 2. as automated as possible so that as properties change, the docs
-   are updated automagically.
+   are updated automatically.
 
 The function `matplotlib.artist.kwdoc` and the decorator
-`matplotlib.docstring.dedent_interpd` facilitate this.  They combine python
+`matplotlib.docstring.dedent_interpd` facilitate this.  They combine Python
```

Comment:
Given the big squak some raised when I tried to uwe this facilty, should this be in the doc guide?

Reply:
I'll still re-iterate that I was told explcitly *not* to do this for the legend docs a month ago.  So if we are not allowed to use this functionality it should be noted here.